### DISCUSSION TITLE
Welcome to eduAlgo Discussions!

### DISCUSSION THREAD

**Discussion by Abhijit2505** [edualgo/eduAlgo#127] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Manoj-Paramsetti** [comment]
@Abhijit2505, Can I add some basic programs in eduAlgo?
I like to add some programs like:
```
Armstrong Number
Anagram of numbers
Towers of Hanoi
Counter Spiral
```

**Reply by Abhijit2505** [reply]
@Manoj-Paramsetti, Check the existing repository once, if something is not present you can add them. 
Two things to note,
- don't add algorithms as a code that is available like codes on the internet, write the code in a functional way so that the user can pass some parameters and use the algorithm
- Don't plagiarize the code, try to add some innovative features like hint(example) etc.
